CorsixTH/Lua/dialogs/place_objects.lua

```diff
@@ -697,7 +697,13 @@ function UIPlaceObjects:setBlueprintCell(x, y)
     end
     if self.object_anim and object.class ~= "SideObject" then
       if allgood then
-        allgood = not world:wouldNonSideObjectBreakPathfindingIfSpawnedAt(x, y, object, self.object_orientation, roomId)
+        if world:wouldNonSideObjectBreakPathfindingIfSpawnedAt(x, y, object, self.object_orientation, roomId) then
+          if self.world.app.config.allow_blocking_off_areas then
```

Comment:
@Alberth289346 I guess this should actually be self.ui.app.config in the 3 instances of this check as I can get it to crash fairly easily as app isn't indexable.

Reply:
:o I'll have a look at it. Thanks